google/cloud/bigquery/client.py

```diff
@@ -515,9 +538,13 @@ def create_routine(
         )
         resource = routine.to_api_repr()
         try:
-            api_response = self._call_api(
-                retry, method="POST", path=path, data=resource, timeout=timeout
-            )
+            span_attributes = {"path": path, "exists_ok": exists_ok}
```

Comment:
Does it make sense to pass in `exists_ok` as an attribute if it's not passed to the actual API? It's only affecting how the client deals with certain response codes.

Reply:
makes sense I can remove this.